lib/cocoapods-packager/builder.rb

```diff
@@ -110,9 +110,16 @@ def copy_headers
       headers_source_root = "#{@public_headers_root}/#{@spec.name}"
 
       Dir.glob("#{headers_source_root}/**/*.h").
-        each { |h| `ditto #{h} #{@fwk.headers_path}/#{h.sub(headers_source_root, '')}` }
-
-      if File.exist?("#{@public_headers_root}/#{@spec.name}/#{@spec.name}.h")
+          each { |h| `ditto #{h} #{@fwk.headers_path}/#{h.sub(headers_source_root, '')}` }
+
+      # if custom 'module_map' is specified add it
+      # to the framework distribution, otherwise check
+      # if a header exists that is equal to 'spec.name'', if so
+      # create a default 'module-map' one using it.
+      if !@spec.module_map.nil?
+        module_map_file = "#{@sandbox_root}/#{@spec.name}/#{@spec.module_map}"
```

Comment:
This is actually a more general issue in the `Builder` class - to not pollute this PR with more general refactoring, I have created a separate issue #80 to address this for the whole class.